examples/common/src/chariott/inspection.rs

```diff
@@ -3,11 +3,9 @@
 
 use std::{borrow::Borrow, collections::HashMap};
 
-use chariott_common::{
-    proto::common::{
-        fulfillment::Fulfillment, inspect_fulfillment::Entry as ProtoEntry, InspectFulfillment,
-    },
-    query::regex_from_query,
+use chariott_common::query::regex_from_query;
+use chariott_proto::common::{
+    fulfillment::Fulfillment, inspect_fulfillment::Entry as ProtoEntry, InspectFulfillment,
```

Comment:
nit: Should we rename `ProtoEntry` to `EntryMessage` for consistency?

Reply:
Done in 44ce11f2310210263841d7709e36adec95269785.